holidays/countries/jordan.py

```diff
@@ -0,0 +1,82 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/python-holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import GREGORIAN_CALENDAR
+from holidays.groups import InternationalHolidays, IslamicHolidays, ChristianHolidays
```

Comment:
Alphabetical order for the imports:
```suggestion
from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
```